testsys/src/delete.rs

```diff
@@ -277,3 +332,50 @@ async fn all_resources_deletion_order(k8s_client: &Client) -> Result<Topological
     }
     Ok(topo_sort)
 }
+
+/// Delete a resource from a testsys cluster by force.
+/// The finalizers will be removed from the resource and the resource will be deleted.
+/// The job for resource deletion will also be deleted.
+/// This should only be used if a resource has already failed to delete.
+async fn force_delete_resource(k8s_client: Client, resource_name: &str) -> Result<()> {
```

Comment:
Can we move this logic to the resource client since it's the client that understands the job and its naming convention?